docs/developer-guide/etendo-copilot/installation.md

```diff
@@ -42,29 +45,33 @@ In addition, you can install only the module containing the **Etendo Copilot** b
 
```

Comment:
I can't comment above, but a new version of the plugin is going to be released, should the version that will be released be updated with the changes in the tasks. Or is it still compatible with previous versions of the plugin?